alacritty_terminal/src/event_loop.rs

```diff
@@ -374,17 +266,10 @@ where
                             }
                         },
 
-                        token
-                            if token == self.pty.read_token()
-                                || token == self.pty.write_token() =>
-                        {
-                            #[cfg(unix)]
-                            if UnixReady::from(event.readiness()).is_hup() {
-                                // Don't try to do I/O on a dead PTY.
-                                continue;
-                            }
+                        token if token == tty::PTY_READ_TOKEN || token == tty::PTY_WRITE_TOKEN => {
+                            // FIXME: once polling supports extra reported modes bring it back.
```

Comment:
This should be an issue. Comments are useless and just get lost.

Also missing what "it" is here. Bring what back?